### ISSUE TITLE
Files Explorer: ToolTip with full path

### ISSUE THREAD

**Issue by Qomaa** [jcaillon/3P#166]
In Files Explorer it would be nice, if a ToolTip would appear when you point with the mouse cursor over a file name. At least in the search files view.
![1](https://cloud.githubusercontent.com/assets/26574474/25694156/71ae1874-30ad-11e7-91a7-e23d0db41208.PNG)

I have multiple files with the same name in different folders and when I search them I can't distinct them and have to open the folder (middle click).

Keep up the great work. I use this plugin every day!

**Comment by jcaillon** [comment]
Yes this is a feature i intended to do for a long time. But.. they are always more urgent matters i guess ^^"
I'll definetly consider it.

Did you try clicking on the magnifying glass to switch the view to FILTER MODE, this will show you in which folder each file is.

br
